data-raw/toy_data.R

```diff
@@ -0,0 +1,11 @@
+# styler: off
+toy_data <- tibble::tribble(
```

Comment:
this is weird - I do not know why this file is here since it was a commit from another branch :(